packages/appium/lib/cli/parser.js

```diff
@@ -280,6 +293,50 @@ class ArgParser {
       }
     }
   }
+
+  /**
+   * Add subcommand and sub-sub commands for 'setup' subcommand.
+   * @param {import('argparse').SubParser} subParser
+   */
+  static _addSetupToParser(subParser) {
+    const setupParser = subParser.add_parser('setup', {
+      add_help: true,
+      help: `Install latest official drivers and/or plugins for specific usage ` +
+        `if APPIUM_HOME has no drivers and plugins. Default is 'mobile' set.`,
+    });
+
+    ArgParser._patchExit(setupParser);
+    const extSubParsers = setupParser.add_subparsers({
+      dest: `setupCommand`,
+    });
+
+    const parserSpecs = [
```

Comment:
I'm not quite sure we need to enforce 'no drivers' limitation. We may just be smart and skip already installed drivers or plugins and only install these that are actually missing

Reply:
Agreed, that would make it a lot simpler for the end user

Reply:
Yea, I thought to relax the behavior later. This was to reduce initial implementation diff at this time

Reply:
Modified to install not existing drivers/plugins in the preset list.